evap/staff/templates/staff_course_comments_section.html

```diff
@@ -0,0 +1,48 @@
+{% load morefilters %}
+{% load i18n %}
+
+{% for group in sections_grouped %}
+    {% for questionnaire, contributor, is_responsible, results in group.list %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <span class="panel-title">{{ questionnaire.public_name }}{% if contributor %}: {{ contributor.full_name }}{% if is_responsible %} (responsible){% endif %}{% endif %}</span>
+            </div>
+            <div class="panel-body">
+                {% for result in results %}
+                    <p>{{ result.question.text }}</p>
+                    <table class="table table-striped">
+                        <thead>
+                            <tr>
+                                <th class="col-sm-1">{% trans "ID" %}</th>
+                                <th class="col-sm-9">{% trans "Comment" %}</td>
+                                <th class="col-sm-2">{% trans "Publish" %}</td>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for answer in result.answers %}
+                                <tr>
+                                    <td>{{ answer.id }}</td>
+                                    <td>
+                                        {% if answer.reviewed_answer %}
+                                            <a class="comment-link" href="{% url "evap.staff.views.course_comment_edit" semester.id course.id answer.id %}">{{ answer.reviewed_answer }}</a><br />
+                                            <span class="comment-original">({{ answer.original_answer }})</span>
```

Comment:
i would prefer an "edit"-button to the answer-as-link, but have no good reasons for that right now. what do you think?